docs/concepts/vega-chain.md

```diff
@@ -85,61 +84,117 @@ Exceptions to automatic nomination:
 ### Un-nominating validators
 Participants can remove their nomination at the end of an epoch, or immediately. The un-nominated tokens will be restored back to the participant's associated token balance.
 
-**Un-nominate towards the end of the epoch** 
-
+#### Un-nominate towards the end of the epoch
 A participant can un-nominate towards the end of the current epoch, which means the stake is not used for the validator from the following epoch. 
 
 The action is announced in the next available block of the same epoch, but the nominator keeps their nomination active until the last block of that epoch. At that point, the tokens are returned. The nominator cannot move those tokens before the epoch ends.
 
```

Comment:
Suggest including: this way a the token holder (and its nominated validator) is still entitled to the rewards from that epoch. using un-delgate now forfeits any rewards from that epoch

Reply:
Add to above: 'un-delegate now or moving the tokens to a different Ethereum adress'

Reply:
I've added most of this, though in a slightly different form. 

Because the repercussions of 'un-nominate/delegate now' are described in that section, I left that as-is.